package-lock.json

```diff
@@ -1613,14 +1613,14 @@
       "dev": true
     },
     "node_modules/@truffle/provider": {
-      "version": "0.2.34",
-      "resolved": "https://registry.npmjs.org/@truffle/provider/-/provider-0.2.34.tgz",
-      "integrity": "sha512-078SPxa6tiRsjxGObhE79Yw26+JNVhub23AArviBPcc5EGkRzDj4Wj5NNKsZIzhK7eFy5deQkc5HtQIAnZngrQ==",
+      "version": "0.2.35",
```

Comment:
Please check if this if package-lock.json update is necessary 
I am not sure as there is no package.json update

Reply:
This was one of the conflicts i was facing when updating the branch